subt/depth2scan.py

```diff
@@ -152,14 +152,15 @@ def adjust_scan(scan, depth_scan):
 class DepthToScan(Node):
     deg30 = int(30*720/270)
 
-    def __init__(self, config, bus):
+    def __init__(self, config, bus, depth_params=DepthParams()):
```

Comment:
one more

Reply:
actually, this is not going to work - there can only be the two params here `config` and `bus` and nothing more or nothing less.